### PR TITLE
[#164763735] Add quota_definition_guid to the billing event consolidation

### PR DESCRIPTION
**Pull request by mogds**
What
----

The paas-admin cost report currently queries the cf api for the quota guid and the
billing endpoint for the billing events.

The quota guid isn't associated with the billing events so the quota in the
report is unrelated to the values. This updates the consolidation to include
the org's quota_definition_guid so the correct quota is avaiable via the
billing endpoint.

- Adds a `quota_definition_guid` to the billing event that's pulled in via a join between orgs and billable_event_components during the consolidation. 
- Adds org data the orgs table as part of the test run.

How to review
-----

- Code Review 
- Run tests

Who can review
-----

Not me or @AP-Hunt

### CODE REVIEW

**Comment by richardTowers** [comment]
Looks like travis gets counterfeiter from the latest thing it can go get, which is broken at the moment. We can 👀on Monday.

File: README.md [hunk 1]
```diff
@@ -288,6 +288,7 @@ curl -s -G -H "Authorization: $(cf oauth-token)" 'http://localhost:8881/billable
 		"org_guid":        "51ba75ef-edc0-47ad-a633-a8f6e8770944",
 		"space_guid":      "276f4886-ac40-492d-a8cd-b2646637ba76",
 		"plan_guid":       "f4d4b95a-f55e-4593-8d54-3364c25798c4",
+                "quota_definition_guid": "dcb680a9-b190-4453-a2d1-cdb1377e42f4",
```
Comment (AP-Hunt) [inline README.md]: Extra whitespace

File: eventstore/store_billable_events.go [hunk 2]
```diff
@@ -158,36 +158,39 @@ func WithBillableEvents(query string, filter eventio.EventFilter, args ...interf
...
+					b.storage_in_mb,
+					b.number_of_nodes,
+					b.duration * filtered_range,
+					b.component_formula
+				) * b.currency_rate) as price_ex_vat
 			from
-				billable_event_components,
-				filtered_range
+			    filtered_range,
+				billable_event_components b
```
Comment (AP-Hunt) [inline eventstore/store_billable_events.go]: Inconsistent indentation

File: eventstore/store_consolidated_billable_events.go [hunk 3]
```diff
@@ -70,6 +70,7 @@ func (e *EventStore) getConsolidatedBillableEventRows(tx *sql.Tx, filter eventio
 			space_guid,
 			space_name,
 			plan_guid,
+			quota_definition_guid,
```
Comment (AP-Hunt) [inline eventstore/store_consolidated_billable_events.go]: Remove empty new line

**Pull request merged**